common/src/Model/GameConfig.h

```diff
@@ -111,8 +111,9 @@ namespace TrenchBroom {
         struct FlagConfig {
             std::string name;
             std::string description;
+            int bitpos;
```

Comment:
Shouldn't this be `size_t`? We usually use that for indices.

Reply:
Preexisting setFlags function taking a QList<int> as input list of bitpositions. I can pull on that thread tho and see where it goes.

Reply:
If setFlags is changed to accept a list of size_t instead, I think the only necessary change outside of what I'm touching here would be IssueBrowser::updateFilterFlags.

This:
```c++
QList<int> flags;
QStringList labels;

for (const Model::IssueGenerator* generator : generators) {
    const Model::IssueType flag = generator->type();
    const std::string& description = generator->description();

    flags.push_back(flag);
    labels.push_back(QString::fromStdString(description));
}
```
would need to be something like this:
```c++
QList<size_t> flags;
QStringList labels;

for (const Model::IssueGenerator* generator : generators) {
    const Model::IssueType flag = generator->type();
    const std::string& description = generator->description();

    flags.push_back(static_cast<size_t>(flag));
    labels.push_back(QString::fromStdString(description));
}
```
...unless I redefine IssueType to be size_t rather than int.

Reply:
Ah, I see. Ok, leave it an int then, it's a bit of a mess with Qt using int indices and us using size_t indices like stl.

Reply:
I'd rather prefer size_t after having thought about it more. We can use QList<int> for the Qt stuff, but in our own models, we should be using size_t for consistency reasons.

Reply:
Got it. Would you prefer the static_cast as above, or changing IssueType to size_t?